src/api/db/data/20200420080753_generate_web_notifications.rb

```diff
@@ -0,0 +1,44 @@
+class GenerateWebNotifications < ActiveRecord::Migration[6.0]
+  def up
+    generate_web_notifications
+    generate_event_subscripitions
+  end
+
+  def down
+    raise ActiveRecord::IrreversibleMigration
+  end
+
+  private
+
+  def generate_web_notifications
+    Notification::RssFeedItem.all.each do |rss_notification|
+      duplicate = rss_notification.dup
+      duplicate.type = 'Notification::WebItem'
+      # to avoid duplicated web notifications
+      Notification::WebItem.find_or_create_by(duplicate.attributes.except('id'))
```

Comment:
:thought_balloon:  Man I still don't like all of this data duplication...

Reply:
The migration has changed and don't duplicate the Notification.